codingtest_ajw/3일차/자릿수-더하기.js

```diff
@@ -0,0 +1,12 @@
+function solution(n)
+{
+    let sum = 0
+    const nString = n.toString()  
+    const nArray = nString.split('')
+    
+    for (let i = 0; i < nArray.length; i++){
+        sum += parseInt(nArray[i])
```

Comment:
```suggestion
        sum += +nArray[i]
```
크게 상관은 없는데 이런 방법도 있다는걸 알려드립니다.